src/geant_argus/geant_argus/templates/htmx/incident/_incident_quick_glance_modal.html

```diff
@@ -1,29 +1,32 @@
 {% load incident_extras %}
-<dialog id="modal_{{ incident.source_incident_id }}" class="modal">
-  <div class="modal-box bg-base-200">
-    <h3 class="text-lg font-bold">Quick Glance</h3>
-    <table class="table table-sm">
-      <tbody>
-        {% for item in incident_description_glance_table %}
-          <tr>
-            <th>{{ item.name }}</th>
-            {% with value=incident|get_quick_glance_item:item %}
-              <td>
-                {% if item.cell_template %}
-                  {% include item.cell_template %}
-                {% else %}
-                  {{ value|default:"-" }}
-                {% endif %}
-              </td>
-            {% endwith %}
-          </tr>
-        {% endfor %}
-      </tbody>
-    </table>
-    <div class="modal-action">
-      <form method="dialog">
-        <button class="btn bg-neutral text-neutral-content">Close</button>
-      </form>
+<dialog id="modal_{{ incident.source_incident_id }}" class="modal" onclick="event.target === this && this.close()">
+    <div class="modal-box bg-base-200">
+        <div class="modal-action">
+            <form method="dialog" class="absolute right-2 top-2">
+                <button class="text-xl font-bold text-neutral hover:text-red-500" aria-label="Close">
+                  <i class="fa-solid fa-xmark text-lg"></i>
+                </button>
+            </form>
+        </div>
+
+        <h3 class="text-lg font-bold">Quick Glance</h3>
```

Comment:
Using absolute positioning is generally not a good practice. I think here you'd wrap the close button `form` and the `h3` in a `div`, and make that a `class=flex` flexbox element. You can then use justify and align to properly position the close button. Added benefit is that the close button will then be positioned at the same hight as the Quick Glance header. 

See  https://v3.tailwindcss.com/docs/flex and https://css-tricks.com/snippets/css/a-guide-to-flexbox/ for information on how to use flexbox

(Also, I see you're wrapping the close button in a ``class=modal-action`` element. ``modal-action`` is a daisyUI class that is meant to be used for buttons at the bottom of a modal, so you should get rid of it here)